src/Items/ItemHandler.cpp

```diff
@@ -251,7 +262,24 @@ cItemHandler * cItemHandler::CreateItemHandler(int a_ItemType)
 		case E_ITEM_SPIDER_EYE:
 		case E_ITEM_STEAK:
 		{
-			return new cItemFoodHandler(a_ItemType);
+			handler.push_back(cpp14::make_unique<cItemFoodHandler>(a_ItemType));
+			break;
+		}
+
+		case E_ITEM_COOKED_FISH:
+		{
+			handler.push_back(cpp14::make_unique<cItemFoodHandler>(a_ItemType, E_META_COOKED_FISH_FISH));
+			handler.push_back(cpp14::make_unique<cItemFoodHandler>(a_ItemType, E_META_COOKED_FISH_SALMON));
+			break;
+		}
+
+		case E_ITEM_RAW_FISH:
+		{
+			handler.push_back(cpp14::make_unique<cItemFoodHandler>(a_ItemType, E_META_RAW_FISH_FISH));
+			handler.push_back(cpp14::make_unique<cItemFoodHandler>(a_ItemType, E_META_RAW_FISH_SALMON));
+			handler.push_back(cpp14::make_unique<cItemFoodHandler>(a_ItemType, E_META_RAW_FISH_CLOWNFISH));
+			handler.push_back(cpp14::make_unique<cItemFoodHandler>(a_ItemType, E_META_RAW_FISH_PUFFERFISH));
```

Comment:
What do you think about subclassing the cItemFoodHandler for each fish, so that the Metavalue doesn't need to be passed and saved?
Or is it better to keep the number of files/classes low?